content/promcon/2016/index.html

```diff
@@ -0,0 +1,110 @@
+        <p>
+          The PromCon Call for Papers (CFP) is open for talk submissions until
+          April 30, 2016, and we encourage you to submit a talk!
+        </p>
+        <p>
+          <a class="btn btn-primary btn-lg" href="https://docs.google.com/forms/d/1b7lq7-h8f0JRPThl6oc3Q70YsbzR__FcGNKbCvdetow/viewform" target="_blank" role="button">Submit Talk</a>
+        </p>
+        <p>
+          We are accepting talk proposals around the following topics, with
+          experience levels ranging from beginner to expert:
+        </p>
+        <p>
+          <ul>
+            <li>Integrating with Prometheus</li>
+            <li>Using Prometheus</li>
+            <li>Prometheus use case reports</li>
+            <li>Prometheus fundamentals and philosophy</li>
+            <li>Prometheus internals and core development</li>
+          </ul>
+        </p>
+        <p>
+          <b>Speaker reimbursements:</b> Since PromCon is a non-commercial
+          community conference, we will likely not be able to reimburse speakers
+          for travel and accommodation. If you want to give a talk but cannot
+          get reimbursed by your company, please submit a proposal anyways and
+          let us know in the "Additional note to the selection committee" field.
+          We will then see what we can do, depending on available funding.
+        </p>
+
+        <h2>Become a sponsor</h2>
+        <p>
+          As a community conference with a minimal entrance fee, we will depend
+          on sponsors for making PromCon possible. We will especially need to
+          cover the following expenses:
+        </p>
+        <p>
+          <ul>
+            <li>Professional video recordings of all talks</li>
+            <li>Catering for 80 attendants</li>
+            <li>An evening after-party</li>
+            <li>Printed materials, banners, lanyards, etc.</li>
+          </ul>
+        </p>
+        <p>
+          Do you want to help make this happen and <b>get your company featured
+          in videos, banners, and printed materials?</b> Please register to be
+          notified below and we will contact you when we have defined detailed
+          sponsorship conditions:
+        </p>
+        <p>
+          <a class="btn btn-primary btn-lg" href="https://docs.google.com/forms/d/17KakZ4dE_8d1D6lyGA9SIE15vAm9NIiOPhlDiVkM9ak/viewform" target="_blank" role="button">Notify Me</a>
+        </p>
+        <p>Thanks for your interest in becoming a sponsor!</p>
+
+        <h2>Register to attend</h2>
+        <p>
+          Registration will open soon — fill out the form linked below to get
+          notified when the registration opens.
+        </p>
+        <p>
+          <a class="btn btn-primary btn-lg" href="https://docs.google.com/forms/d/12H6o3vvnnQUJSDV1mQ77EqpU9B64Jk70iFboHwb2y3M/viewform" target="_blank" role="button">Notify Me</a>
+        </p>
+        <p>
+          Admission will likely cost around $50, but we will provide exceptions
```

Comment:
We'll probably need this higher, it's better to come down in price than go up so I'd put it nearer €100.